src/util.ts

```diff
@@ -26,6 +26,7 @@ export function getDefaultRules(): BsLintConfig['rules'] {
         'anon-function-style': 'auto',
         'aa-comma-style': 'no-dangling',
         'type-annotations': 'off',
+        'interface-type': 'info',
```

Comment:
This should be `'off'` by default. Developers will need to opt-in to this feature.

Reply:
DOne